kolibri/core/assets/src/views/k-select/index.vue

```diff
@@ -142,7 +142,12 @@
   .k-select-inline
     display: inline-block
     vertical-align: bottom
-    width: 100px
     margin-right: 16px
 
+  .language-list-dropdown.k-select-inline
```

Comment:
We do not want make any changes to the k-select. Instead we want to add this style to the language switcher component. Also you accidentally checked in some strings.

Reply:
I already moved it @christianmemije. And also removed some strings that's included in the PR.